README.md

```diff
@@ -24,6 +24,7 @@
 
 ## Learning About Music
 * [learningmusic.ableton.com](https://learningmusic.ableton.com)
+* [learningsynths.ableton.com](https://learningsynths.ableton.com)
```

Comment:
I could also see this going in the "Synths" section, but I put it here since that's the spot the original issue recommended.

Reply:
Good callout. This seems like the right place to put it